### ISSUE TITLE
Add content and logo to parallax inner box

### ISSUE THREAD

**Issue by codetheorem** [vinayaksh42/smvdu-codeclub#3]
## Description
Add our logo and line "A community of developers of Smvdu".Add more styling to it like border-radius,color,etc.

## Mocks
![image](https://user-images.githubusercontent.com/54779517/82591451-fbddde80-9bbc-11ea-8969-72d2ec13ccd5.png)

### Update 
- [x] It should be within our designing lines.
## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

**Comment by gauravchauhan2403** [comment]
assign this to me. I will work on it.

**Comment by RohitDhatrak** [comment]
@codetheorem @vinayaksh42 
Is this issue still assigned?
If not I would like to work on it.

**Comment by codetheorem** [comment]
@RohitDhatrak go ahead.

**Comment by RohitDhatrak** [comment]
@codetheorem you want me to replace the logo and the tagline and add parallax effect to it right?

**Comment by codetheorem** [comment]
@RohitDhatrak Yes like in the image given above.you can also apply your creativity .

**Comment by RohitDhatrak** [comment]
@codetheorem Where can I get your logo?
